assets/src/utils/date-helper.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * Get DayJS supported date format.
+ *
+ * @since WEPOS_LITE_SINCE
+ *
+ * @param {string} dateFormat The date format to convert
+ *
+ * @return {string} DayJS supported date format
+ */
+export function wepos_get_dayjs_date_format( dateFormat = wepos.wp_date_format ) {
+    let formatMap = {
+        // Day
+        d: 'DD',
+        D: 'ddd',
+        j: 'D',
+        // Month
+        F: 'MMMM',
+        m: 'MM',
+        M: 'MMM',
+        n: 'M',
+        // Year
+        Y: 'YYYY',
+        y: 'YY'
+    }
+
+    let i = 0;
+    let char = '';
+    let dayJsDateFormat = '';
+
+    for ( i = 0; i < dateFormat.length; i++ ) {
+        char = dateFormat[i];
+
+        if ( char in formatMap ) {
+            dayJsDateFormat += formatMap[char];
+        } else {
+            dayJsDateFormat += char;
+        }
```

Comment:
Hope so, this is enough instead of these lines -

```js
dateRangePickerFormat += formatMap[char] ? formatMap[char] : char;
```

If it's OK, please change respective places below.